### PR TITLE
Add featured artists to music tag

### PR DESCRIPTION
**Pull request by Frnot**
Featured artists are currently ignored when generating tags. This information can be extracted from the "performers" field of the metadata and appended to the artist field in separate tags.

This solution for extracting featured artists is not as perfect as I would like, but I am building a list of testcases to validate against.

### CODE REVIEW

File: qobuz_dl/metadata.py [hunk 1]
```diff
@@ -138,9 +158,12 @@ def tag_flac(
...
     artist_ = d.get("performer", {}).get("name")  # TRACK ARTIST
     if istrack:
-        audio["ARTIST"] = artist_ or d["album"]["artist"]["name"]  # TRACK ARTIST
+        artists = [artist_ or d["album"]["artist"]["name"]]  # TRACK ARTIST
     else:
-        audio["ARTIST"] = artist_ or album["artist"]["name"]
+        artists = [artist_ or album["artist"]["name"]]
+
+    artists.extend(_get_featured_artists(d))
+    audio["ARTIST"] = artists
```
Comment (vitiko98) [inline qobuz_dl/metadata.py:166]: Are lists supported?

Comment (Frnot) [reply qobuz_dl/metadata.py:164]: Yes. see: https://mutagen.readthedocs.io/en/latest/user/gettingstarted.html#multiple-values or https://xiph.org/vorbis/doc/v-comment.html#implications for an example in the FLAC tag spec

Comment (vitiko98) [reply qobuz_dl/metadata.py:169]: Sadly there's no way to know the exact format of the performers string. We'll just be guessing and may end with broken tags at any time. 

I wouldn't risk such a vital tag as ARTIST. If you find any other FLAC tag to suit your PR let me know.

Comment (Frnot) [reply qobuz_dl/metadata.py:169]: Unfortunately, this is the nature of things when the world cannot decide on a standard way to tag music.

**Pull request closed without merging**